src/pages/items/glasses/index.js

```diff
@@ -60,7 +60,7 @@ function Glasses(props) {
                     return (
                         <div className="center-content">
                             {t('Status')}
-                            <div>{t('Turn/Ergo')}</div>
+                            <>{t('Turn/Ergo')}</>
```

Comment:
This for layout too
![B1089401-B772-4DC8-9517-DA7F81B691D0](https://user-images.githubusercontent.com/10927011/185233312-7705da7d-c867-4cf3-ad27-ad7c48abba91.jpeg)